external/webserver/http_client.c

```diff
@@ -314,15 +322,27 @@ int http_parse_message(char *buf, int buf_len, int *method, char *url,
 							len->content_len += cha;
 						}
 
-						response->total_len = len->content_len;
 						*body = buf + len->sentence_start + 5;
 						buf[buf_len + len->message_len - 2] = '\0';
 
+						if (response->entity) {
+							response->entity_len = strlen(response->entity);
+						}
 						HTTP_LOGD("buf_len : %d len->sentence_start : %d len->message_len : %d len->content_len %d\n", buf_len, len->sentence_start, len->message_len, len->content_len);
 						read_finish = true;
 					}
 					len->message_len += buf_len;
 
+					response->total_len = len->content_len;
+					response->entity_len = buf_len;
+
+					if(is_header == true) {
+						response->entity_len = 0;
+						HTTP_LOGD("[HEADER_BODY]buf_len : %d len->sentence_start : %d len->message_len : %d len->content_len %d\n", buf_len, len->sentence_start, len->message_len, len->content_len);
+					}
+
+					entity = *body;
+					HTTP_LOGD("[BODY]buf_len : %d len->sentence_start : %d len->message_len : %d len->content_len %d\n", buf_len, len->sentence_start, len->message_len, len->content_len);
```

Comment:
What is your standard for HTTP_LOGD, nvdbg and printf?

Reply:
The webclient module uses nvdbg,
http_client moduel uses HTTP_LOGD.
printf is used to display messages that should be printed on the screen rather than debug messages.